client/my-sites/stats/stats-subscribers-chart-section/index.tsx

```diff
@@ -175,16 +175,42 @@ export default function SubscribersChartSection( {
 			{ ! isChartLoading && chartData.length !== 0 && (
 				<>
 					<div className="subscribers-section-legend" ref={ legendRef }></div>
-					<UplotChart
-						data={ chartData }
-						legendContainer={ legendRef }
-						period={ period }
-						// Use variable --studio-jetpack-green for chart colors on Odyssey Stats.
-						mainColor={ isOdysseyStats ? '#069e08' : undefined }
-						fillColorFrom={ isOdysseyStats ? 'rgba(6, 158, 8, 0.4)' : undefined }
-						fillColorTo={ isOdysseyStats ? 'rgba(6, 158, 8, 0)' : undefined }
-						yAxisFilter={ hideFractionNumber }
-					/>
+					{ isChartLibraryEnabled && (
+						<LineChart
+							chartData={ [
+								{
+									label: 'Subscribers',
+									options: {
+										stroke: '#069e08',
```

Comment:
We should have use different color for Calypso

Reply:
Yup absolutely. On my current PR list to add a Calypso color theme to the chart library, which we can use webpack in Odyssey to replace it with the Jetpack theme.